### ISSUE TITLE
Wont let me pick random

### ISSUE THREAD

**Issue by drago5212** [DougDougGithub/ChatGodApp#15]
so, I got it to connect to my twitch account, and connected to azure aswell but when testing it, it wont allow me to pick a random user even if people do !player1 2 or 3, i am not getting any errors so im just confused on whats going on

**Comment by jessiejs** [comment]
this has no support so yeah dont expect a response

**Comment by drago5212** [comment]
Yeah i know but at the time of posting there was alot of dougs community on so i was seeing if anyone one of them will reply with a fix or atleast some help. it was mainly for the community not doug himself
